src/Imath/ImathFrustumTest.h

```diff
@@ -191,12 +211,6 @@ FrustumTest<T>::setFrustum (const Frustum<T>& frustum, const Matrix44<T>& camera
     cameraMatrix = cameraMat;
 }
 
-////////////////////////////////////////////////////////////////////
-// isVisible(Sphere)
-// Returns true if any part of the sphere is inside
-// the frustum.
-// The result MAY return close false-positives, but not false-negatives.
-//
 template <typename T>
```

Comment:
Is there a reason that these comments disappeared?

Reply:
They moved to the declaration inside the class. The convention (such as it is) is for comments to appear in the class declaration but not be repeated by the method implementation.